src/interfaces/document-interface.ts

```diff
@@ -0,0 +1,10 @@
+import { DocumentTypes } from '@/enums'
```

Comment:
I expect this interface will be used when we are able to fetch a list of documents from the API (soon).